### ISSUE TITLE
Passworded Pages form

### ISSUE THREAD

**Issue by paulxb** [gantry/gantry5#2155]
My Wordpress site is set to use its own folder as described here: https://codex.wordpress.org/Giving_WordPress_Its_Own_Directory
When I add a passworded page, the password form doesn't redirect to the subfolder.
eg when I enter a password it goes to domain.com/wp-login.php?action=postpass (404 error)
I think it should go to domain.com/wordpress/wp-login.php?action=postpass

**Comment by newkind** [comment]
Can you please try to edit the file `wp-content/themes/<your-theme>/views/partials/password-form.html.twig` and change :
`{{ site.url }}`
to :
`{{ site.link }}`
Did this fixed the issue?

**Comment by paulxb** [comment]
That didn't fix the problem.

**Comment by newkind** [comment]
Did you clear cache after modifying the file?

**Comment by paulxb** [comment]
Yes, cleared the cache and proved by hard coding the path, which works. 
If it is helpful, my Wordpress Address is: domain.com/wordpress
Site Address: domain.com
